creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -12,9 +12,6 @@ const CreatorNode = require('@audius/libs/src/services/creatorNode')
 const SecondarySyncHealthTracker = require('./secondarySyncHealthTracker')
 const { generateTimestampAndSignature } = require('../apiSigning')
 
-// Maximum number of time to wait for a sync operation, 6 minutes by default
-const MaxSyncMonitoringDurationInMs = 360000 // ms
```

Comment:
this value of 6 min was wayy too large for local testing, so i moved to envvar with lower val for local dev (10s)